tests/fsharp/Compiler/Language/StringInterpolation.fs

```diff
@@ -618,8 +618,17 @@ let xd = $"%A{}" // empty expression
 """
         CompilerAssert.TypeCheckWithErrorsAndOptions  [| "--langversion:preview" |]
             code
-            [|(FSharpErrorSeverity.Error, 10, (2, 15, 2, 17),
-               "Unexpected interpolated string (final part) in binding")
+            [|(FSharpErrorSeverity.Error, 3382, (2, 15, 2, 15),
+               "Empty interpolated string fill begun at or before here")
+            |]
+
+        let code = """
+let xd = $"%A{  }" // empty expression
+"""
+        CompilerAssert.TypeCheckWithErrorsAndOptions  [| "--langversion:preview" |]
+            code
+            [|(FSharpErrorSeverity.Error, 3382, (2, 15, 2, 17),
+               "Empty interpolated string fill begun at or before here")
```

Comment:
is that a proper sentence?

Reply:
The "fill" terminology probably isn't familiar to the user. 

We are using similar here:
```
3380,parsEofInInterpolatedVerbatimString,"Incomplete interpolated verbatim string begun at or before here"
3381,parsEofInInterpolatedTripleQuoteString,"Incomplete interpolated triple-quote string begun at or before here"
3382,parsEmptyFillInInterpolatedString,"Empty interpolated string fill begun at or before here"
```

Reply:
Perhaps something like this:

```
3382,parsEmptyFillInInterpolatedString,"Invalid interpolated string. This interpolated string expression fill is empty, an expression was expected."

Reply:
Yep that's my point. All of those are not really user friendly

Reply:
This was something I brought up in our code review. It can be addressed independently though